packages/helpermodules/hardware_configuration.py

```diff
@@ -16,5 +17,13 @@ def get_hardware_configuration_setting(name: str):
         return json.loads(f.read())[name]
 
 
+def get_serial_number() -> str:
+    try:
+        with open("/home/openwb/snnumber", "w") as file:
+            return file.read().replace("\n", "")
```

Comment:
Hier wird die Datei zum Schreiben ("w") geöffnet und dann nur daraus gelesen?
```suggestion
        with open("/home/openwb/snnumber", "r") as file:
            return file.read().replace("\n", "")
```